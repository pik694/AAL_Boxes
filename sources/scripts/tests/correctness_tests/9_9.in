9
9 91 72
43 78 63
22 75 91
7 55 43
72 46 80
86 42 47
72 52 70
29 33 33
96 67 3

