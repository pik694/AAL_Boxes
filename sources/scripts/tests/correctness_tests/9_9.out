3 0 8 7 5 2 1 4 6 
1184656->925218
